test/functional/helpers/consoleLogger/index.js

```diff
@@ -1,28 +1,113 @@
-const createConsoleLogger = (context, logLevel) => {
-  const t = context;
-  const level = logLevel;
-  let cursor = 0;
-
-  const getNewMessages = async () => {
```

Comment:
I think Martin's E2E test is using this `getNewMessages` helper and that's why it's failing in CI.